monorepo/website/src/pages/docs/how-to/use-tools-and-add-new-ones.mdx

```diff
@@ -0,0 +1,79 @@
+
+We are interested in adding new useful external tools to this menu. If you are a tool developer interested in having your tool featured here, please read on.
+
+### Requirements
+
+For a tool to be integrated into the Pathoplexus tools menu, it must be able to:
+
+- Accept file URLs as parameters in its web address
+- Automatically fetch and process the files from these URLs and do something useful with them
+
+When users select your tool from the Tools menu, Pathoplexus will generate download URLs, configured in a way that you specify, then redirect users to your tool with these URLs embedded in the web address. Your tool needs to parse these URLs from its incoming request and retrieve the files directly from Pathoplexus.
+
+### Adding your tool to Pathoplexus
+
+The list of tools is defined in the `linkOuts` section of the `values.yaml`
+configuration of Pathoplexus.
+
+A link-out entry has the following structure:
+
+```json
+{
+  "name": "My Tool",
+  "url": "http://example.com/analysis?data={{[unalignedNucleotideSequences]}}",
+  "maxNumberOfRecommendedEntries": 100
+}
+```
+
+`name` is displayed in the menu, `url` is a template that may contain
+placeholders and `maxNumberOfRecommendedEntries` is optional. When this value is
+set, users will be warned if they attempt to send more sequences than the limit
+suggests.
+
+### URL template placeholders
+
+The URL template may contain placeholders wrapped in square brackets. They are
+replaced with download links generated for the current search results. Available
+placeholders are:
+
+- `[unalignedNucleotideSequences]` – link to the unaligned FASTA file.
+- `[alignedNucleotideSequences]` – link to the aligned FASTA file.
+- `[metadata]` – link to a metadata table in TSV format.
+
+Each placeholder can be extended with options:
+
+- `:[segment]` – restricts the sequence download to a specific segment, e.g.
+  `[unalignedNucleotideSequences:S]`.
+- `+rich` – include rich FASTA headers when supported, e.g.
+  `[unalignedNucleotideSequences+rich]`.
+- `|format` – request a specific data format such as `json`, e.g.
+  `[metadata|json]`.
+- For metadata, `+col1,col2` can be used to include only
+  certain fields, e.g. `[metadata+country,host]`.
+
+To ensure parts of the URL are properly URL encoded, wrap them in double curly
+braces. For example `{{my query}}` becomes `my%20query` after processing. This
+encoding happens after any placeholders are substituted, so `{{[unalignedNucleotideSequences]}}` will become a properly URL-encoded URL to access the unaligned nucleotide sequences. You can even doubly-nest `{{}}` syntax where necessary.
+
+Multiple placeholders can appear in a single URL, allowing you to pass both
+sequence and metadata downloads to an external tool.
+
+You can create a PR in the [Pathoplexus repository](https://github.com/pathoplexus/pathoplexus) to add your link-out entries to the configuration under a specific organism, or raise an issue to suggest this.
```

Comment:
```suggestion
You can create a PR or issue in the [Pathoplexus repository](https://github.com/pathoplexus/pathoplexus) to suggest adding your link-out entries to the configuration under a specific organism, and we'll consider all suggestions.
```
```suggestion
You can create a PR or issue in the [Pathoplexus repository](https://github.com/pathoplexus/pathoplexus) to suggest adding your link-out entries to the configuration under a specific organism, and we'll consider all suggested new tools.
```

Reply:
Oh no why did it appear twice.

Anyway - basically my point here was to clarify that just adding a PR doesn't necessarily mean we'll include your tool - people may come up with silly things we don't want to be obligated to include.